sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoSchemaTranslator.java

```diff
@@ -126,6 +127,7 @@
  * </ul>
  */
 @SuppressWarnings({
+  "nullness", // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
```

Comment:
Could you address the null checker errors rather than suppressing it in this way? The goal of BEAM-10402 is to reduce the number of these suppressions over time :)

Reply:
Ok, I modified the code to use an empty Schema as a placeholder in the map, instead of ```null```. Also, I added a unit test to show that the placeholder empty Schema is not the same as an empty proto object, so it won't accidentally throw an exception if an empty Schema is passed in to the ```getSchema``` function.